ansible_collections/arista/cvp/molecule/cv_image_v3/test_cv_image_v3.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Test cv_image_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    # Make sure given image does not exist on CVP
+    image_name: "EOS-4.25.4M.swi"
+  tasks:
+
+  ### Run all the tasks sequentially ###
+
+    - name: "Create bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: add
+        bundle_name: Test_bundle
+        image_list:
+          - "{{image_name}}"
```

Comment:
looks like we also need to add the TerminAttr extension to this otherwise it's failing with
```yaml
TASK [Create bundle cv_server] *************************************************
fatal: [cv_server]: FAILED! => changed=false
  msg: 'POST: https://cvp:443/web/image/saveImageBundle.do : Request Error: SWI does not contain a supported TerminAttr version. A separate supported TerminAttr extension is required.'
```

so I've updated that to

```yaml
    - name: "Create bundle {{inventory_hostname}}"
      arista.cvp.cv_image_v3:
        mode: bundle
        action: add
        bundle_name: Test_bundle
        image_list:
          - "{{image_name}}"
          - "{{extension_name}}"
```

Reply:
Fixed!